core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -453,6 +454,17 @@ private void setSelectedInstance(InstanceFactory instanceFactory) {
     getPropertyPanel().setSelectedInstance(instanceFactory);
   }
 
+  private void triggerOrthographicCameraChange(Expression selectedFieldExpression) {
```

Comment:
This is called on camera and model selection and that seems to work in general.
If the active model is moved or resized it can end up outside of the view.
Not sure if the additional tracking is worth it, but I observed the effect and want you to be aware of it.